Patterns/CancelRequest.md

```diff
@@ -2,8 +2,14 @@
 
 When long-running or expensive API calls are called by clients which have a tendency to cancel requests, an API controller should implement a cancellation token which is triggered when the client has cancelled the request. This prevents the API from doing additional work for a client which does not want to have the result anymore.
 
-## Web Apps (.NET 5 & .NET Core 3.1)
+## Web Apps
 
 - Controller: Add `CancellationToken token` as parameter to accept the cancellation token in the controller method.
 - Pass the token to all subsequent async calls. 
-- Optional: Catch the `TaskCanceledException` to log the cancellation.
\ No newline at end of file
+- Optional: Catch the `TaskCanceledException` to log the cancellation.
+
+## Function apps
+
+- Function: Add `CancellationToken token` as parameter to accept the cancellation token in the function.
```

Comment:
Are there any entrypoints where you don't just add the cancellation token as the (last) parameter? If not might as well just consolidate this to one section. Otherwise we are gonna end up with 99% the same text for all other types of application or endpoint (eg. minimal apis)